WcaOnRails/spec/mailers/job_failure_mailer_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe JobFailureMailer, type: :mailer do
+  describe "notify_admin_of_job_failure" do
+    let(:job) { Delayed::Job.new(id: 42) }
+    let(:exception) do
+      begin
+        raise "error!"
+      rescue Exception => e
+        return e
+      end
+    end
+    let(:mail) { JobFailureMailer.notify_admin_of_job_failure(job, exception) }
```

Comment:
Ah, I like that solution for this test! I'm going to continue to use a real exception with a real backtrace for http://localhost:3000/rails/mailers/job_failure_mailer/notify_admin_of_job_failure, though, just to get a better sense of what the generated emails will look like.